modules/compiler/vecz/source/transform/packetizer.cpp

```diff
@@ -202,13 +204,24 @@ class Packetizer::Impl : public Packetizer {
   ///
   /// @return Packetized instruction.
   Value *packetizeGroupBroadcast(Instruction *I);
-  /// @brief Returns true if the instruction is a subgroup shuffle.
+  /// @brief Returns true if the instruction is any subgroup shuffle.
   ///
   /// @param[in] I Instruction to query.
   ///
-  /// @return True if the instruction is a call to a mux subgroup shuffle
+  /// @return The group collective data if the instruction is a call to a mux
+  /// subgroup shuffle-like builtin - std::nullopt otherwise.
```

Comment:
might be useful to give some indication of what a shuffle like instruction is (although is perhaps defined somewhere else).

Reply:
I don't want to be overly specific as it's an implementation detail and ultimately depends on the mux spec.

Reply:
you mean the mux spec specifies what a shuffle-like builtin is?

Reply:
It specifies a bunch of `shuffle*` builtins, yeah